src/Microsoft.Azure.SignalR.Common/ServiceConnections/ServiceConnectionContainerBase.cs

```diff
@@ -31,7 +30,7 @@ internal abstract class ServiceConnectionContainerBase : IServiceConnectionConta
 
         private volatile ServiceConnectionStatus _status;
 
-        private volatile bool _terminated = false;
+        protected volatile bool _terminated = false;
```

Comment:
>protected [](start = 8, length = 9)

private is enough?

Reply:
No, we can't simply call `base.ShutdownAsync` since `StrongContainer` has its own `on-demand` connections and needs to be stopped after the `_terminated` flag to be set to `true`.